Source/Core/VideoCommon/TextureDecoder.h

```diff
@@ -88,3 +88,8 @@ void TexDecoder_SetTexFmtOverlayOptions(bool enable, bool center);
 /* Internal method, implemented by TextureDecoder_Generic and TextureDecoder_x64. */
 void _TexDecoder_DecodeImpl(u32* dst, const u8* src, int width, int height, int texformat,
                             const u8* tlut, TlutFormat tlutfmt);
+
+static constexpr int DXTBlend(int v1, int v2)
```

Comment:
Since this is in a header, `static` should be dropped (constexpr implies inline, so there's no need to specify inline to resolve linkage in this case).